common/PlayAreaScreen.tsx

```diff
@@ -158,6 +195,13 @@ export function PlayAreaScreen() {
         showHint: false,
       },
     });
+    if (gameModule.gameId == 'checkers' && checkersFlag) {
```

Comment:
what is this?

Reply:
checkers moves are build from two acts of the user (choose tool & move the tool) 
and I want the sound will appear only when it moves and not when the player choose his tool